app/javascript/components/admin/UserModal.js

```diff
@@ -12,12 +12,12 @@ class UserModal extends React.Component {
       roleTitle: this.props.initialUserData.role_title ? this.props.initialUserData.role_title : this.props.roles[0],
       isAPIEnabled: this.props.initialUserData.is_api_enabled ? this.props.initialUserData.is_api_enabled : false,
       isLocked: this.props.initialUserData.is_locked ? this.props.initialUserData.is_locked : false,
+      notes: this.props.initialUserData.notes ? this.props.initialUserData.notes : '',
```

Comment:
I think this can be simplified to:
```suggestion
      notes: this.props.initialUserData.notes || '',
```

Reply:
same with all above it as well except role title

Reply:
Resolved in https://github.com/SaraAlert/SaraAlert/pull/805/commits/6692b2c0557b7786044f99801d4bce25b73e6047